File: fizzbuzz.py
"""Totes a program"""

def fizzbuzz(n):
    """Does the FizzBuzz thing n times"""
    for x in range(1, n+1):
        if x % 5 == 0 and x % 3 == 0:
            print("FizzBuzz")
        elif x % 3 == 0:
            print("Fizz")
        elif x % 5 == 0:
            print("Buzz")
        else:
            print(x)

if __name__ == "__main__":
    fizzbuzz(30)

File: shitty-java-programs/fizzbuzz.java
class fizzbuzz {
    
    public static void print(String thing) {
        System.out.println(thing);
    }

    public static void main(String[] args) {
        for (int i = 0; i<=100; i++) {
            String s = "";
            if ((i % 3) == 0) {
                s = s + "Fizz";
            }
            if ((i % 5) == 0) {
                s = s + "Buzz";
            }
            if (((i % 3) != 0) && ((i % 5) != 0)) {
                s = "" + i;
            }
            print(s + "");
        }

    }
}

// adjlsad
// jksdlsad
// klhasd